filebeat/docs/fields.asciidoc

```diff
@@ -59,6 +59,7 @@ grouped in the following categories:
 * <<exported-fields-mongodb>>
 * <<exported-fields-mssql>>
 * <<exported-fields-mysql>>
+* <<exported-fields-mysql>>
```

Comment:
You'll want to name this something like `exported-fields-mysql-enterprise`, it conflicts with the ID right above it.